### PR TITLE
use current time as endTime for aggregate queries and 

### PR DESCRIPTION
**Pull request by gsps1**
using empty list/map when groupBy, tags or time-range is not provided

Jira link : https://issues.cask.co/browse/CDAP-1955

### CODE REVIEW

**Comment by anwar6953** [comment]
In comment, you mean 'is not provided', right?

**Comment by gsps1** [comment]
yeah. fixed it now.

File: cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java [hunk 1]
```diff
@@ -387,7 +387,7 @@ public void testAggregateQueryBatch() throws Exception {
     QueryRequestFormat query1 = new QueryRequestFormat(getTagsMap("namespace", "yourspace", "app", "WCount1",
                                                                   "flow", "WCounter", "flowlet", "splitter"),
                                            ImmutableList.of("system.reads"), ImmutableList.<String>of(),
-                                           ImmutableMap.of("aggregate", "true"));
+                                           ImmutableMap.<String, String>of());
```
Comment (abaranau) [inline cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java]: why? also: do we still provide timeRange as map?

Comment (gsps1) [inline cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java]: just to test it defaults to aggregate=true, when nothing is provided

**Comment by abaranau** [comment]
LGTM

**Pull request merged**